### DISCUSSION TITLE
NSM logs size & indices

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#9880] [General]
I am trying to limit the /nsm size, but doesn't working
I create lifecycle policy, but it doesn't seem to work, how do I check it?
my hot phase -> maximum size 50 giga & maximum age 30 days 
warm phase -> 5 days
Cold phase -> 5 days
Delete phase -> 5 days

![image](https://user-images.githubusercontent.com/29997801/222426691-08277caf-cffb-4f00-8e45-14fed167cbdf.png)

Another question, why my indices look like that?
![image](https://user-images.githubusercontent.com/29997801/222427078-f84b402a-e68d-40df-a9e1-149f2bffbeaf.png)

**Comment by mahmoudelhaj** [comment]
Index lifecycle policies apply to index templates
![image](https://user-images.githubusercontent.com/29997801/222427605-b425f6ae-7fde-451f-abf8-24ad957a6814.png)

**Comment by cm-ops** [comment]
Would you provide some more information about you grid https://github.com/Security-Onion-Solutions/securityonion/discussions/1720?

**Comment by mahmoudelhaj** [comment]
1) can limit nsm logs size using lifecycle management  ??
2) my indices should look like so-firewall-3-3-2023 ?? Why do they appear differently ??
![image](https://user-images.githubusercontent.com/29997801/222775139-b6456df5-7165-47d4-81c1-164643df836a.png)

**Comment by cm-ops** [comment]
1. https://docs.securityonion.net/en/latest/curator.html?curator-vs-index-lifecycle-management-ilm#curator-vs-index-lifecycle-management-ilm We use Curator to manage the indices to ensure you don't run out of disk space. 
2. You are looking at the raw data on disk. If you use the REST API to query the indices you will see `so-firewall-* ` you can use the `so-elasticsearch-query _cat/indices` command to see them in the `so-*` list.
